.github/workflows/build_docs.yaml

```diff
@@ -0,0 +1,53 @@
+name: docs
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+jobs:
+  build_sphinx_docs:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          # Need to clone everything for the git tags.
+          fetch-depth: 0
+
+      - name: Set up Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: '3.10'
+          cache: "pip"
+          cache-dependency-path: "setup.cfg"
+
+      - name: Install sqlite
+        run: sudo apt-get install sqlite libyaml-dev
+
+      - name: Update pip/wheel infrastructure
+        run: |
+          python -m pip install --upgrade pip
+          pip install wheel
+
+      - name: Install postgresql (server)
```

Comment:
You don't need these or the ones below to build the docs -- those are for butler optional classes.